logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/FlowManagerServiceImpl.kt

```diff
@@ -59,6 +59,18 @@ actual class FlowManagerServiceImpl(
         }
     }
 
+    override suspend fun flipToFrontCamera(conversationId: ConversationId) {
+        withContext(dispatcher) {
+            flowManager.setVideoCaptureDevice(conversationId.toString(), "front")
```

Comment:
maybe we can add this as a const?

Reply:
We will not need it later, it's only used internally inside the function